dockers/docker-fpm-frr/docker_init.sh

```diff
@@ -3,18 +3,46 @@
 mkdir -p /etc/frr
 mkdir -p /etc/supervisor/conf.d
 
-CFGGEN_PARAMS=" \
-    -d \
-    -y /etc/sonic/constants.yml \
-    -t /usr/share/sonic/templates/supervisord/frr_vars.j2 \
-    -t /usr/share/sonic/templates/supervisord/supervisord.conf.j2,/etc/supervisor/conf.d/supervisord.conf \
-    -t /usr/share/sonic/templates/bgpd/bgpd.conf.j2,/etc/frr/bgpd.conf \
-    -t /usr/share/sonic/templates/zebra/zebra.conf.j2,/etc/frr/zebra.conf \
-    -t /usr/share/sonic/templates/staticd/staticd.conf.j2,/etc/frr/staticd.conf \
-    -t /usr/share/sonic/templates/frr.conf.j2,/etc/frr/frr.conf \
-    -t /usr/share/sonic/templates/isolate.j2,/usr/sbin/bgp-isolate \
-    -t /usr/share/sonic/templates/unisolate.j2,/usr/sbin/bgp-unisolate \
-"
+MGMT_FRAMEWORK_CONFIG=`sonic-cfggen -d -v 'DEVICE_METADATA["localhost"]["frr_mgmt_framework_config"]'`
+CRIT_PROC_BACKUP="/usr/local/frr/supervisord/critical_processes.bak"
```

Comment:
can you turn the critical_process as a j2 template, generate the critical_process list based on the MGMT_FRAMEWORK_CONFIG?

Reply:
Yes, will add j2 template to generate critical process list

Reply:
Addressed